docs/installation.rst

```diff
@@ -1,6 +1,8 @@
-============
-Installation
-============
+.. _installation::
+
+=================
+Installing ndcube
+=================
 
 ndcube requires Python 3.5+, SunPy 0.9+, astropy and matplotlib.
```

Comment:
SunPy 0.9+? also what version of Astropy - surely a newer one >4.0?

Reply:
Great catch!